backend/sirius-web-spring-collaborative/src/main/java/org/eclipse/sirius/web/spring/collaborative/projects/EditingContextEventProcessor.java

```diff
@@ -362,10 +354,16 @@ private void disposeRepresentation(UUID representationId) {
 
     @Override
     public void dispose() {
+        this.logger.trace(MessageFormat.format("Disposing the editing context \"{0}\"", this.editingContext.getId())); //$NON-NLS-1$
+
         this.executor.shutdown();
 
-        this.representationEventProcessors.values().stream().forEach(IRepresentationEventProcessor::dispose);
+        this.representationEventProcessors.values().stream().forEach(entry -> {
+            entry.getDisposable().dispose();
+            entry.getRepresentationEventProcessor().dispose();
```

Comment:
`RepresentationDescriptionEntry` could have a `dispose()` itself which does this.